examples/tutorials/AlphafoldXDeepChem.ipynb

```diff
@@ -0,0 +1,1626 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    
</pre><p>Make sure to remove extra whitespace</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>